### DISCUSSION TITLE
Remove no longer needed work around for Firefox added in Suit CSS

### DISCUSSION THREAD

**Discussion by mornir** [tailwindlabs/tailwindcss#4724] [Ideas]
Hey!
On Discord, Adam was wondering if this [work around for Firefox](https://github.com/tailwindlabs/tailwindcss/blob/ea3892fc9167cece4a1d3aab2c342bf791baea7e/src/plugins/css/preflight.css#L33) was still necessary.

I did some research about it. Here are my findings:
- The work around was [added in 2014](https://github.com/suitcss/base/commit/88853f550c7cb421852f9ad0aa70e896e7e92f82).
- In this issue thread, a [link to a bug reproduction on plnkr](http://plnkr.co/edit/lCcQce46VJE1NWVMHJV6) is shared.
- Using Firefox 44, I was able to confirm the bug with the plnkr reproduction (but not with [my own code reproduction on Codepen](https://codepen.io/mornir0/pen/bGqJMXQ)).
- Using the latest Firefox version (v90), I was unable to reproduce the bug, neither on plunkr or on codepen. It seems that the bug was fixed, but I couldn't find a ticket on Bugzilla. (It would be really surprising that such a bug still exists in 2021...)

Using heuristic, modern browsers are now smarter when it comes to applying focus rings. With this (no longer needed) work around, Tailwind doesn't let the browser decide when it comes to buttons. Tailwind users need to use `focus-visible` to get that heuristic back. 

In my understanding, removing that workaround would solve this issue. `focus-visible` can still be used for styling better looking focus rings.

**Comment by adamwathan** [comment]
Converted this to an issue so I don't lose track of it:

https://github.com/tailwindlabs/tailwindcss/issues/4779
